thread/common/thrprof.c

```diff
@@ -409,13 +428,26 @@ omrthread_get_user_time(omrthread_t thread)
 	 *
 	 */
 
-	FILETIME creationTime, exitTime, kernelTime, userTime;
-	int64_t totalTime;
+	FILETIME creationTime;
+	FILETIME exitTime;
+	FILETIME kernelTime;
+	FILETIME userTime;
+	int64_t totalTime = 0;
+
+	memset(&creationTime, 0, sizeof(creationTime));
+	memset(&exitTime, 0, sizeof(exitTime));
+	memset(&kernelTime, 0, sizeof(kernelTime));
+	memset(&userTime, 0, sizeof(userTime));
 
 	/* WARNING! Not supported on Win95!  Need to test to ensure this fails gracefully */
 
 	if (GetThreadTimes(thread->handle, &creationTime, &exitTime, &kernelTime, &userTime)) {
-		totalTime = ((int64_t)userTime.dwLowDateTime | ((int64_t)userTime.dwHighDateTime << 32));
+		ULARGE_INTEGER userTimeU64;
+		memset(&userTimeU64, 0, sizeof(userTimeU64));
```

Comment:
Please ensure there's a blank line before each group of `memset()` calls for separation from the declarations; here and before lines 722, 770, and 1124.

Reply:
fixed